.ci/deploy.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -eo pipefail # Exit with nonzero exit code if anything fails
+
+SOURCE_BRANCH="master"
+TARGET_BRANCH="gh-pages"
+
+# Pull requests and commits to other branches shouldn't try to deploy, just build to verify
+if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_BRANCH" != "$SOURCE_BRANCH" ]; then
```

Comment:
Taking into account the `ENCRYPTION_LABEL` decision described below, the remaining case is

- pushes to a branch on https://github.com/coala/gci-leaders should not deploy to "gh-pages"
- pushes to a branch on https://github.com/jayvdb/gci-leaders should deploy to "gh-pages"

Thinking a bit broader, an additional case exists on the `community` project, because I want forks of `community` to be able to have their own network of forks. (without requiring conflicting changes) i.e.

- pushes to a branch on https://github.com/librehealthio/community should not deploy to "gh-pages"

The one way to achieve this is detecting that https://github.com/coala and https://github.com/librehealthio are orgs vs users, and give them different default behaviours.

A simpler way, is require that either orgs or users set a variable to choose different behaviour.  I think the easiest is the require that orgs do nothing, but a user needs to set `SOURCE_BRANCH="*"` , or something similar, to explicitly say I want to deploy any branch I push, overwriting stuff is OK.